src/java.base/share/classes/java/io/Reader.java

```diff
@@ -391,6 +391,41 @@ public int read(char[] cbuf) throws IOException {
      */
     public abstract int read(char[] cbuf, int off, int len) throws IOException;
 
+
+    /**
+     * Reads all remaining characters into a string.
+     *
+     * <p> This method reads all content including the line separators in
+     * the middle and/or at the end. The resulting string will contain line
+     * separators as they appear in the original content. The method does not
+     * close this reader nor its underlying stream. If an I/O error occurs,
+     * the states of the reader and its underlying stream are unspecified.
+     *
+     * @apiNote
+     * This method is intended for simple cases where it is appropriate and
+     * convenient to read all lines into a String. It is not intended for
+     * reading a large number of lines.
+     *
+     * @return     a String containing all lines
+     *
+     * @throws     IOException       If an I/O error occurs
+     * @throws     OutOfMemoryError  If the content is extremely large,
+     *                               for example larger than {@code 2GB}
+     *
+     * @see java.nio.file.Files#readString
+     *
+     * @since 25
+     */
+    public String readString() throws IOException {
+        StringBuilder result = new StringBuilder();
```

Comment:
```java
StringBuilder result = new StringBuilder().append('\u4e2d').delete(0, 1);
```
Of course, we can also create a StringBuilder with coder as UTF16 in this way, but such code smells bad.